webapp/components/Registration/CreateUserAccount.vue

```diff
@@ -1,117 +1,107 @@
 <template>
-  <ds-container width="small">
-    <ds-card v-if="success" class="success">
-      <ds-space>
-        <sweetalert-icon icon="success" />
-        <ds-text align="center" bold color="success">
-          {{ $t('registration.create-user-account.success') }}
-        </ds-text>
-      </ds-space>
-    </ds-card>
-    <ds-card v-else class="create-account-card">
-      <client-only>
-        <locale-switch />
-      </client-only>
-      <ds-space centered>
-        <img
-          class="create-account-image"
-          alt="Create an account for Human Connection"
-          src="/img/sign-up/nicetomeetyou.svg"
-        />
-      </ds-space>
-      <ds-space>
-        <ds-heading size="h3">
-          {{ $t('registration.create-user-account.title') }}
-        </ds-heading>
-      </ds-space>
+  <div v-if="response === 'success'">
+    <sweetalert-icon icon="success" />
```

Comment:
it would be nice to wrap these in `<transition name="ds-transition-fade">` to keep it consistent across the code base